.travis.yml

```diff
@@ -1,12 +1,13 @@
 language: python
-python:
-  - 2.7
-  - 3.3
-  - 3.4
+env:
+  - TOXENV=py26
```

Comment:
@Ichimonji10 Sure!  It's easier to just have travis just run `tox`.  However, if you want travis to run a job for each supported python version, we'll need to duplicate the `TOXENV` in `.travis.yml`.  I presume that's how you'll want the results, since it's consistent with how tests were previously run.

I'll update the `.travis.yml` per your feedback.